x/upgrade/client/cli/tx.go

```diff
@@ -16,10 +17,13 @@ import (
 )
 
 const (
-	FlagUpgradeHeight = "upgrade-height"
-	FlagUpgradeInfo   = "upgrade-info"
-	FlagNoValidate    = "no-validate"
-	FlagDaemonName    = "daemon-name"
+	FlagUpgradeHeight    = "upgrade-height"
+	FlagUpgradeInfo      = "upgrade-info"
+	FlagNoValidate       = "no-validate"
+	FlagNoValidateInfoCS = "no-validate-info-checksum"
+	FlagNoValidateBinCS  = "no validate-binary-checksums"
+	FlagNoValidateCS     = "no-validate-checksums"
```

Comment:
can we have only one flag for checksum? I think it's enough to have the last one.

Reply:
I split them thinking of Provenance's specific use case where we want checksums on the binaries, but don't want one on the url to get the JSON. By having it split out, it provides the most flexibility.